mlflow/metrics/utils/prompts/v1.py

```diff
@@ -0,0 +1,94 @@
+from dataclasses import dataclass, field
+from typing import Any, Dict, List
+
+from mlflow.metrics.base import (
+    EvaluationExample,
+)
+from mlflow.metrics.utils.prompt_template import (
+    PromptTemplate,
+)
+
+default_model = "openai:/gpt4"
+default_parameters = {
+    "temperature": 0.0,
+    "max_tokens": 100,
+    "top_p": 1.0,
+}
+grading_system_prompt_template = PromptTemplate(
+    """
+    Please act as an impartial judge and evaluate the quality of the provided output which
+    attempts to produce output for the provided input based on a provided information.
+    You'll be given a function grading_function which you'll call for each provided information,
+    input and provided output to submit your justification and score to compute the {name} of
+    the output.
+
+    Input:
+    {input}
+
+    Provided output:
+    {output}
+
+    {variables}
+
+    Metric definition:
+    {definition}
+
+    Grading criteria:
+    {grading_prompt}
+
+    {examples}
+
+    And you'll need to submit your grading for the {name} of the output,
+    using the following format:
+    Reasoning for {name}: [your step by step reasoning about the {name} of the output]
+    Score for {name}: [your score number between 0 to 4 for the {name} of the output]
```

Comment:
Has @suanmiao approved this template?

Reply:
I am using the https://github.com/databricks/doc_qa/blob/b0ecd4486b784ecbb6047e9bedb2e4a545ed5cbc/databricks/labs/doc_qa/variables/doc_qa_template_variables.py template from @suanmiao